infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryService.kt

```diff
@@ -160,9 +160,9 @@ class GeometryService @Autowired constructor(
         return geometryDao.getLinkingSummaries(planIds)
     }
 
-    fun getDuplicateGeometryPlanHeader(newFile: InfraModelFile): GeometryPlanHeader? {
```

Comment:
logiikka noissa fetch vs get metodeissa on ollut että fetch käy kannassa (tyypillisesti vain dao-metodi) kun taas get on kaikki muut, erityisesti ylemmän tason / yhdistelevät.

Reply:
Hyvä huomio. 
Jäin itse asiassa vielä miettimään, että pitäisikö oikeastaan käyttää sanaa find koska miellän itse ainakin niin, että get palauttaa aina not nullin.

Reply:
Vähän ehkä laajemmin mulla ainakin menee nämä fetchien ja gettien väliset erot aina sekaisin, koska mulle Geoviitteen ulkopuolisessa maailmassa ne on käytännössä synonyymejä keskenään. Vaikka tuo Jyrkin sanoma jaottelu onkin sinällään ihan ymmärrettävä, niin en itse sinällään tiedä kuinka merkityksellistä on oikeasti erotella noita - Dao-tasolla olevan hakumetodin voinee olettaa käytännössä aina käyvän kannassa ennen pitkää (kun ei meillä dataa oikein muuallakaan tallessa ole), ja Service-tasolla mikään ei taas koskaan saisi käydä kannassa, koska kyseessä ei ole Dao.